### DISCUSSION TITLE
Upgrade React Admin to V4

### DISCUSSION THREAD

**Discussion by GreenMachine01** [amplication/amplication#4366] [Ideas] [answered]
Do you think we should pursue this? Why?

https://marmelab.com/react-admin/
https://marmelab.com/react-admin/Upgrade.html

**Comment by yuval-hazaz** [comment, marked as answer]
From reading their upgrade guide it seems like the upgrade will not be difficult - we are mainly using their core API, and I do not expect major breaks. 
so, cost vs advantage - my answer is yes

**Comment by overbit** [comment]
Yes please!
